### PR TITLE
Fixing up docker containers

### PR DESCRIPTION
**Pull request by ewels**
* Switch from quay.io back to dockerhub.
* Stop automatically trying to add the container tag - instead just have a variable that needs editing next to the pipeline version.

### CODE REVIEW

**Comment by apeltzer** [comment]
I guess that the code works fine - Travis CI just doesn't build because the dockerhub image wasn't built before ....

**Comment by ewels** [comment]
Yup - just trying to set it up now but dockerhub is being a buggy piece of crap.

File: conf/base.config [hunk 1]
```diff
@@ -15,7 +15,7 @@ process {
 
   // Variable to specify the docker / singularity image tag to use
   // Picks up on use of -r <version> in nextflow command
```
Comment (MaxUlysse) [inline conf/base.config]: I guess we can remove this comment too

Comment (ewels) [reply conf/base.config]: haha, yeah I guess I should have spotted that ;)

**Comment by ewels** [comment]
That was more tweaks needed than expected! Tests now passing @apeltzer @MaxUlysse 👍

File: README.md [hunk 2]
```diff
@@ -2,9 +2,10 @@
 
 [![Build Status](https://travis-ci.org/nf-core/methylseq.svg?branch=master)](https://travis-ci.org/nf-core/methylseq)
 [![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A50.27.6-brightgreen.svg)](https://www.nextflow.io/)
+[![Gitter](https://img.shields.io/badge/gitter-%20join%20chat%20%E2%86%92-4fb99a.svg)](https://gitter.im/nf-core/Lobby)
 
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](http://bioconda.github.io/)
-[![Docker Repository on Quay](https://img.shields.io/badge/docker-available-green.svg "Docker Repository on Quay")](https://quay.io/repository/nf-core/methylseq)
+[![Docker Container available](https://img.shields.io/docker/automated/nf-core/methylseq.svg)](https://hub.docker.com/r/nf-core/methylseq/)
```
Comment (MaxUlysse) [inline README.md]: I think it should be `nfcore` for all docker related links...

Comment (ewels) [reply README.md]: Well spotted 👍

**Pull request merged**